docs/docs/01-ibc/06-channel-upgrades.md

```diff
@@ -11,6 +11,20 @@ slug: /ibc/channel-upgrades
 Learn how to upgrade existing IBC channels.
 :::
 
+## Initializing a Channel Upgrade
+
+A channel upgrade is initialised by submitting the `ChanUpgradeInit` message, which can be submitted only by the chain itself upon governance authorization. This message should specify an appropriate timeout window for the upgrade.
+
+As part of the handling of the `ChanUpgradeInit` message, the application's callbacks `OnChanUpgradeInit` will be triggered as well.
+
+After this message is handled successfully, the channel's upgrade sequence will be incremented. This upgrade sequence will serve as a nonce for the upgrade process to avoid error receipt proof collisions.
+
+### Governance Gating on `ChanUpgradeInit`
+
+A message signer who has been designated as the `authority` of the `IBCKeeper` sending the `MsgChanUpgradeInit`. If this proposal passes, the counterparty channels will upgrade by default.
```

Comment:
There is a single channel version being upgraded per `MsgChanUpgradeInit` right?

```suggestion
A message signer who has been designated as the `authority` of the `IBCKeeper` sending the `MsgChanUpgradeInit`. If this proposal passes, the counterparty's channel will upgrade by default.
```